notebook/static/base/js/keyboard.js

```diff
@@ -404,8 +408,8 @@ define([
          *  The shortcut error should be explicit here, because it will be
          *  seen by users.
          */
-        try
-        {
+        var that = this;
```

Comment:
I might be missing something but it doesn't look like `that` is being used anywhere in this function?